org-formation/075-security-hub/security-hub-suppress-infra.yaml

```diff
@@ -369,12 +369,15 @@ Resources:
               - 'aws-foundational-security-best-practices/v/1.0.0/IAM.6'
               - 'arn:aws:securityhub:::ruleset/cis-aws-foundations-benchmark/v/1.2.0/rule/2.7'
               - 'cis-aws-foundations-benchmark/v/1.4.0/3.7'
-              # SecHub Recommendation: no security group allows unrestricted ingress access to port 22.
+              # SecHub Recommendation: no security group allows unrestricted ingress access to port 22/3389.
               # Supression Reason: Allow for bastian hosts and testing
               - 'cis-aws-foundations-benchmark/v/1.2.0/rule/4.1'
```

Comment:
Not sure about this one, I don't see it in findings when I search but there are still others with the same format so leaving for now.